packages/gotop/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/xxxserxxx/gotop
+TERMUX_PKG_DESCRIPTION="A terminal based graphical activity monitor inspired by gtop and vtop"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@medzikuser"
+_MAJOR_VERSION=4.1.2
+# Use the ~ deb versioning construct in the future:
+TERMUX_PKG_VERSION=1:${_MAJOR_VERSION}
```

Comment:
Why you are setting epoch for the package previously unavailable in Termux?

When to use epoch: https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#downgrading-the-package-or-changing-versioning-scheme

Reply:
I was basing it on golang and simply did not delete it but changed it to 1